### DISCUSSION TITLE
SQL Server Agent?

### DISCUSSION THREAD

**Discussion by MarkFergusonTN** [dbeaver/dbeaver#13921] [Q&A]
All, very new to DBeaver. Have installed the Lite edition and liking it for the most part. However, I'm missing any of the SQL Server administration bits that I'm used to seeing in SQL Server Management Studio such as SQL Server Agent Jobs, Security, etc. Am I just missing this option, do I need a version other than Lite, or is this just functionality that does not exist in DBeaver?

![image](https://user-images.githubusercontent.com/41969932/133540897-a09f6b78-68cf-42ca-afa3-2fed421fcfcd.png)

**Comment by LonwoLonwo** [comment]
Hello @MarkFergusonTN 

What DBeaver version do you use?

![2021-09-17 18_23_16-Window](https://user-images.githubusercontent.com/45152336/133809027-bd778dc8-eea5-44bd-84ce-af0fe5542197.png)

I can see the Security group in my last Lite Version.

But we do not have Jobs, yes.

I added a ticket about this feature in the next releases: https://github.com/dbeaver/dbeaver/issues/1651

**Reply by MarkFergusonTN** [reply]
Ahhh, I changed the Navigator view and am now able to see Security as well. That is a shame about the Jobs part. I don't need to see them every day, it would just be nice to be able to commit to one tool. I'm going to keep playing with DBeaver and see if it'll fit my needs 95% of the  time.

Thank you for your help!
